align/schema/constraint.py

```diff
@@ -528,23 +530,25 @@ def group_block_name(cls, value):
 
     def translate(self, solver):
         # Non-zero width / height
+        instances = get_instances_from_hacked_dataclasses(self)
         bb = solver.bbox_vars(self.name)
         yield bb.llx < bb.urx
         yield bb.lly < bb.ury
-        # Grouping into common bbox
-        for b in solver.iter_bbox_vars(self.instances):
-            yield b.urx <= bb.urx
-            yield b.llx >= bb.llx
-            yield b.ury <= bb.ury
-            yield b.lly >= bb.lly
-        instances = get_instances_from_hacked_dataclasses(self)
-        for b in solver.iter_bbox_vars((x for x in instances if x not in self.instances)):
-            yield solver.Or(
-                b.urx <= bb.llx,
-                bb.urx <= b.llx,
-                b.ury <= bb.lly,
-                bb.ury <= b.lly,
-            )
+        # Skip constraints checking during PnR
```

Comment:
@kkunal1408 , what is the intention here? Why skipping constraint check?

Reply:
Previously groupblock constraints used to be removed after their realization. In this PR the group block constraints are retained and for all the identified hierarchies new groupblock constraints are added. The verilog.json file dumped by the compiler will have these for reusability but while reading back this file for PnR, the instances defined in the groupblock constraint no more exist in the design that's why skipped.

Reply:
done